macros/multiple_databases/create_index.sql

```diff
@@ -0,0 +1,8 @@
+{%- macro create_index(model) -%}
+
+{%- if target.type == 'sqlserver' -%}
+    drop index if exists [{{ var("schema_sources") }}].[{{ model }}].[cci];
+    create clustered columnstore index cci on [{{ var("schema_sources") }}].[{{ model }}];
+{%- endif -%}
+
+{%- endmacro -%}
```

Comment:
- We discussed the implementation about adding the database in the logic. This seems 'safer' to me, although it seems to work also without specifying the database. To implement it with database, I found that we can use the following syntax (described here: **https://docs.microsoft.com/en-us/sql/t-sql/statements/drop-index-transact-sql?view=sql-server-ver16)**:

```
drop index if exists cci on {{ this.database }}.{{ var("schema_sources") }}.{{ model }};
create clustered columnstore index cci on {{ this.database }}.{{ var("schema_sources") }}.{{ model }};
```
- Syntax comment: the square brackets are redundant (it is not broken, but also not needed).
- Just a small comment about terminology; model is the dbt terminology for the .sql files we have in the models folder, where each creates a table in the database. This creates an index on the source data, which is not part of the dbt project.